### PR TITLE
add a few more reactions

### PR DESCRIPTION
**Pull request by sleepbot-zapp**

### CODE REVIEW

File: reactions.json [hunk 1]
```diff
@@ -17,7 +17,7 @@
     "Cl / uv": "2-chloropropane",
     "Br / uv": "2-bromopropane"
   },
-  "chloromethane": {
+  "1-chloromethane": {
```
Comment (CodeWithSwastik) [inline reactions.json]: chloromethane should be preferred over 1-chloromethane since it's redundant to write that 1

File: reactions.json [hunk 2]
```diff
@@ -29,7 +29,7 @@
     "AgF": "fluromethane",
     "CH3ONa": "methoxymethane"
   },
-  "bromomethane": {
+  "1-bromomethane": {
```
Comment (CodeWithSwastik) [inline reactions.json]: bromomethane should be preferred over 1-bromomethane since it's redundant to write that 1

File: reactions.json [hunk 3]
```diff
@@ -41,7 +41,7 @@
     "AgF": "fluromethane",
     "CH3ONa": "methoxymethane"
   },
-  "chloroethane": {
+  "1-chloroethane": {
```
Comment (CodeWithSwastik) [inline reactions.json]: chloroethane should be preferred over 1-chloroethane since it's redundant to write that 1

File: reactions.json [hunk 4]
```diff
@@ -54,15 +54,15 @@
     "AgF": "fluroethane",
     "CH3ONa": "methoxyethane"
   },
-  "bromoethane": {
+  "1-bromoethane": {
```
Comment (CodeWithSwastik) [inline reactions.json]: bromoethane should be preferred over 1-bromoethane since it's redundant to write that 1

File: reactions.json [hunk 5]
```diff
@@ -97,24 +97,37 @@
     "Na": "sodium methoxide",
     "SOCl2": "chloromethane",
     "PCl5": "chloromethane",
-    "ZnCl2/ HCl": "chloromethane"
+    "PCl3": "chloroethane",
+    "ZnCl2 / HCl": "chloromethane",
+    "HCOOH": "methyl methanoate",
+    "Cu / 573K": "methanal",
```
Comment (CodeWithSwastik) [inline reactions.json]: Temperature, Pressure conditions should be written inside () while different reagenets should be separated with /

File: reactions.json [hunk 6]
```diff
@@ -97,24 +97,37 @@
     "Na": "sodium methoxide",
     "SOCl2": "chloromethane",
     "PCl5": "chloromethane",
-    "ZnCl2/ HCl": "chloromethane"
+    "PCl3": "chloroethane",
```
Comment (CodeWithSwastik) [inline reactions.json]: Shouldn't this be chloromethane?

**Review by CodeWithSwastik — CHANGES_REQUESTED** [review]

File: reactions.json (hunk 6 above)
Comment (sleepbot-zapp) [reply reactions.json]: yea typo

**Comment by sleepbot-zapp** [comment]
all issues resolved

**Review by CodeWithSwastik — APPROVED** [review]

**Pull request merged**